pkgs/data/icons/bibata-cursors/default.nix

```
@@ -0,0 +1,41 @@
+{ fetchFromGitHub, gksu, gnome-themes-extra, inkscape, pkgs, stdenv, xcursorgen }:
```

```suggestion
{ fetchFromGitHub, gnome-themes-extra, inkscape, stdenv, xcursorgen }:
```
Now that you don't need to scope `gksu` anymore. And `pkgs` is not used